neuromation/api/parsing_utils.py

```diff
@@ -33,11 +56,13 @@ def __post_init__(self) -> None:
 
     def as_docker_url(self) -> str:
         if _is_in_neuro_registry(self):
-            name = f"https://{self.registry}/{self.owner}/{self.name}"
+            name = f"{self.registry}/{self.owner}/{self.name}"
             tag = f":{self.tag}" if self.tag else ""
             return name + tag
         else:
-            return str(self)
+            name = self.name
+            tag = f":{self.tag}" if self.tag else ""
+            return name + tag
```

Comment:
It is common in both branches.

Reply:
good point. I made the changes to simplify the code in the future and forgot about it. Updated